pkg/operator2/operator.go

```diff
@@ -161,11 +165,14 @@ func NewAuthenticationOperator(
 	kubeadminNameFilter := operator.FilterByNames("kubeadmin")
 	configNameFilter := operator.FilterByNames("cluster")
 	prefixFilter := getPrefixFilter()
+	operandNSFilter := operator.FilterByNames("openshift-authentication")
 
 	return operator.New("AuthenticationOperator2", c,
 		operator.WithInformer(routeInformer, targetNameFilter),
 		operator.WithInformer(coreInformers.Services(), targetNameFilter),
 		operator.WithInformer(kubeInformersNamespaced.Apps().V1().Deployments(), targetNameFilter),
+		operator.WithInformer(kubeInformersNamespaced.Core().V1().ServiceAccounts(), targetNameFilter),
+		operator.WithInformer(kubeInformersNamespaced.Core().V1().Namespaces(), operandNSFilter),
 
```

Comment:
I added informer for sa and ns, but looking at our other operators (i looked at service-ca-operator, openshift-apiserver-operator, cluster-kube-controller-manager-operator), AFAICT they don't add the informer for ClusterRoleBindings()  I want to ask why before I add it here.

Reply:
That's a good question, I do not know. I can see that those attempt to use kube-auto-reconciliation, which we could perhaps use here, too, I only need to see how it works.

Reply:
added staticresourcecontroller instead : )